src/bci_build/package/appcollection.py

```diff
@@ -0,0 +1,46 @@
+"""Application Collection Containers that are generated with the BCI tooling"""
+
+from dataclasses import dataclass
+
+from bci_build.package import ApplicationStackContainer
+from bci_build.package import Arch
+
+
+@dataclass
+class ApplicationCollectionContainer(ApplicationStackContainer):
+    """Containers for the Rancher Application Collection Distribution Platform."""
```

Comment:
Will we not publish language stack containers to the application collection?

Reply:
what are you suggesting? calling it "ApplicationCollectionApplicationContainer" so that we have a "ApplicationCollectionDevelopmentContainer" when the need arises? so far no development container has been requested.

Reply:
If we will be publishing everything to the application catalog, then it would make sense to have a flag in the container instead of creating subclasses imho

Reply:
yeah, I was thinking about creating a RegistryMixIn for this but I'm slowly losing patience and energy to do bigger architectural changes on the codebase. 

I think we are not intending to move everything over in one go, and some of the content might remain in two places. but as the descriptions and other labels are not the same, we need to generate them twice.